cli/delete.go

```diff
@@ -0,0 +1,71 @@
+package cli
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/pkg/errors"
+	datapb "go.viam.com/api/app/data/v1"
+)
+
+// DeleteBinaryData deletes binary data matching filter.
+func (c *AppClient) DeleteBinaryData(dst string, filter *datapb.Filter) error {
+	if err := c.ensureLoggedIn(); err != nil {
+		return err
+	}
+	resp, err := c.sendDeleteBinaryDataByFilterRequest(filter)
+	if err != nil {
+		return errors.Wrapf(err, "received error from server")
+	}
+	deletedCount := resp.GetDeletedCount()
+	status := resp.GetResult().GetStatus()
+
+	switch status {
+	case datapb.Status_STATUS_PARTIAL_SUCCESS:
```

Comment:
We can still have deleted some messages with a partial successes, so would still instead the deletedCount.
I think this section could be:

```
if status == datapb.Status_STATUS_PARTIAL_SUCCESS {
    fmt.Fprint(c.c.App.Writer, "errors", resp.GetResult().GetMessage())
}
fmt.Fprintf(c.c.App.Writer, "deleted %d files", deletedCount)
return nil
```

Same for tabular

Reply:
true, done!